generate.go

```diff
@@ -71,51 +81,62 @@ func (ti *typeInfo) generate(filepath string, configType string) (string, error)
 
 	/* Get description, author and references from the file */
 	var fInfo fileInfo
-	yamlFile, err := ioutil.ReadFile(filepath)
+	fileContent, err := ioutil.ReadFile(filepath)
 	if err != nil {
 		return "", err
 	}
-	err = yaml.Unmarshal(yamlFile, &fInfo)
-	if err != nil {
-		return "", err
-	}
-	if fInfo.Author != "" {
-		ti.Author = fInfo.Author
-	}
-	if len(fInfo.References) > 0 {
-		ti.References = fInfo.References
-	}
+	if configType != "data_files" {
+		err = yaml.Unmarshal(fileContent, &fInfo)
+		if err != nil {
+			return "", err
+		}
 
-	if fInfo.Description != "" {
-		ti.Description = fInfo.Description
-	}
+		if fInfo.Author != "" {
+			ti.Author = fInfo.Author
+		}
 
-	if fInfo.Labels != nil {
-		ti.Labels = fInfo.Labels
+		for i := range fInfo.Data {
+			fInfo.Data[i].DestFile = fmt.Sprintf("data_files/%s/%s", ti.Author, fInfo.Data[i].DestFile)
+		}
 
-		// var tags_to_keep = []string{"service", "type"}
-		// for _, v := range tags_to_keep {
-		// 	if x, ok := fInfo.Labels[v]; ok {
-		// 		ti.Tags = append(ti.Tags, x)
-		// 	}
-		// }
-	}
+		if !skipDataFiles {
+			downloadDataFiles(fInfo.Data...)
+		}
 
-	if configType == "collections" {
-		if len(fInfo.Parsers) > 0 {
-			ti.Parsers = fInfo.Parsers
+		if len(fInfo.References) > 0 {
+			ti.References = fInfo.References
 		}
-		if len(fInfo.PostOverflows) > 0 {
-			ti.PostOverflows = fInfo.PostOverflows
+
+		if fInfo.Description != "" {
+			ti.Description = fInfo.Description
 		}
-		if len(fInfo.Scenarios) > 0 {
-			ti.Scenarios = fInfo.Scenarios
+
+		if fInfo.Labels != nil {
+			ti.Labels = fInfo.Labels
+
+			// var tags_to_keep = []string{"service", "type"}
```

Comment:
remove commented lines please

Reply:
This was present before the PR, hence  didn't touch it. Are you sure we should remove this ?